src/app/[locale]/projects/detail/[id]/components/Obligations/ObligationsView/LicenseObligation.tsx

```diff
@@ -0,0 +1,210 @@
+                                    <li key={licenseId} style={{ display: 'inline' }}>
+                                        <Link href={`/licenses/${licenseId}`} className='text-link'>
+                                            {licenseId}
+                                        </Link>
+                                        {index >= licenseIds.length - 1 ? '' : ', '}{' '}
+                                    </li>
+                                )
+                            })}
+                        </ul>
+                    }
+                </div>
+            ),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.releases',
+            name: t('Releases'),
+            formatter: (releases: LicenseObligationRelease[]) => {
+                const previewString = releases.map((r) => `${r.name ?? ''} ${r.version ?? ''}`).join(', ').substring(0, 10)
+                return _(
+                    <ExpandableList releases={releases} previewString={previewString} />
+                )
+            },
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.status',
+            name: t('Status'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.type',
+            name: t('Type'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.id',
+            name: t('Id'),
+            sort: true,
+        },
+        {
+            id: 'licenseObligation.comment',
+            name: t('Comment'),
+            sort: true,
+        }
+    ]
+
+    const initServerPaginationConfig = (session: Session) => {
+        return {
+            url: `${SW360_API_URL}/resource/api/projects/${projectId}/licenseObligations`,
+            then: (data: ProjectObligationsList) => {
+                const tableRows = []
+                for(const [key, val] of Object.entries(data.licenseObligations)) {
+                    const row = []
+                    row.push(
+                        ...[
+                            {
+                                id: key.split(' ').join('_'),
+                                infoText: val.text,
+                            },
+                            key,
+                            val.licenseIds ?? [],
+                            val.releases ?? [],
+                            Capitalize(val.status ?? ''),
+                            val.type ?? '',
+                            val.id ?? '',
+                            val.comment ?? ''
+                        ]
+                    )
+                    tableRows.push(row)
+                }
+                return tableRows
+            },
+            total: (data: ProjectObligationsList) => data.page.totalElements,
+            headers: { Authorization: `Bearer ${session.user.access_token}` },
+        }
+    }
+
+    return (
+        <>
+            {
+                status ? 
```

Comment:
This condition always true. It should be
```suggestion
                (status === 'authenticated') ? 
```